packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/start-job-run.ts

```diff
@@ -0,0 +1,681 @@
+   * Sets values in the spark-env.sh file.
+   *
+   * For more info:
+   * @see https://spark.apache.org/docs/latest/configuration.html#environment-variables
+   *
+   * @returns 'spark-env'
+   */
+  static readonly SPARK_ENV = new Classification('spark-env');
+
+  /**
+   * Sets values in the hive-site.xml for Spark.
+   *
+   * @returns 'spark-hive-site'
+   */
+  static readonly SPARK_HIVE_SITE = new Classification('spark-hive-site');
+
+  /**
+   * Sets values in the log4j.properties file.
+   *
+   * For more settings and info:
+   * @see https://github.com/apache/spark/blob/master/conf/log4j.properties.template
+   *
+   * @returns 'spark-log4j'
+   */
+  static readonly SPARK_LOG4J = new Classification('spark-log4j');
+
+  /**
+   * Sets values in the metrics.properties file.
+   *
+   * For more settings and info:
+   * @see https://github.com/apache/spark/blob/master/conf/metrics.properties.template
+   *
+   * @returns 'spark-metrics'
+   */
+  static readonly SPARK_METRICS = new Classification('spark-metrics');
+
+  /**
+   * Creates a new Classification, can be extended to support a classification
+   *
+   * @param classificationStatement A literal string in case a new EMR classification is released, if not already defined.
+   */
+  constructor(public readonly classificationStatement: string) { }
+}
+
+/**
+ * A configuration specification to be used when provisioning virtual clusters,
+ * which can include configurations for applications and software bundled with Amazon EMR on EKS.
+ *
+ * A configuration consists of a classification, properties, and optional nested configurations.
+ * A classification refers to an application-specific configuration file.
+ * Properties are the settings you want to change in that file.
+ * @see https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-configure-apps.html
+ */
+export interface ApplicationConfiguration {
+
+  /**
+   * The classification within a configuration.
+   *
+   * Length Constraints: Minimum length of 1. Maximum length of 1024.
+   */
+  readonly classification: Classification;
+
+  /**
+   * A list of additional configurations to apply within a configuration object.
+   *
+   * Array Members: Maximum number of 100 items.
+   *
+   * @default - No other configurations
+   */
+  readonly nestedConfig?: ApplicationConfiguration[];
+
+  /**
+   * A set of properties specified within a configuration classification.
+   *
+   * Map Entries: Maximum number of 100 items.
+   *
+   * @default - No properties
+   */
+  readonly properties?: { [key: string]: string };
+}
+
+/**
+ * Configuration setting for monitoring.
+ */
+export interface Monitoring {
+
+  /**
+   * Enable logging for this job.
+   *
+   * If set to true, will automatically create a Cloudwatch Log Group and S3 bucket.
+   * This will be set to `true` implicitly if values are provided for `logGroup` or `logBucket`.
+   *
+   * @default false - logging is enabled by providing values for `logGroup` or `logBucket`
```

Comment:
I was marking comments as resolved without replying if I was incorporating your suggestions. But I guess that might be confusing. 
What you said is correct, and I changed the last line of the documentation to `@default - true if values are provided for `logGroup` or `logBucket`, false otherwise`